From 8728e54845a01940d060f0764adf96bb5eea6d01 Mon Sep 17 00:00:00 2001
From: Fabio Valentini <decathorpe@gmail.com>
Date: Tue, 17 Jan 2023 20:40:56 +0100
Subject: [PATCH] ignore doctest with missing send_wrapper dependency

---
 src/marker.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/marker.rs b/src/marker.rs
index f159d89..c4bc4af 100644
--- a/src/marker.rs
+++ b/src/marker.rs
@@ -70,7 +70,7 @@
 //! This will "work" to smuggle Python references across the closure, because we're not actually
 //! doing anything with threads:
 //!
-//! ```rust, no_run
+//! ```rust, ignore
 //! use pyo3::prelude::*;
 //! use pyo3::types::PyString;
 //! use send_wrapper::SendWrapper;
-- 
2.39.0

