--- pyo3-0.15.2/Cargo.toml	1970-01-01T00:00:01+00:00
+++ pyo3-0.15.2/Cargo.toml	2022-05-05T09:14:10.358610+00:00
@@ -57,42 +57,6 @@
     "docsrs",
 ]
 
-[[bench]]
-name = "bench_call"
-harness = false
-
-[[bench]]
-name = "bench_err"
-harness = false
-
-[[bench]]
-name = "bench_dict"
-harness = false
-
-[[bench]]
-name = "bench_gil"
-harness = false
-
-[[bench]]
-name = "bench_list"
-harness = false
-
-[[bench]]
-name = "bench_pyclass"
-harness = false
-
-[[bench]]
-name = "bench_pyobject"
-harness = false
-
-[[bench]]
-name = "bench_set"
-harness = false
-
-[[bench]]
-name = "bench_tuple"
-harness = false
-
 [dependencies.anyhow]
 version = "1.0"
 optional = true
@@ -109,11 +73,11 @@
 optional = true
 
 [dependencies.indexmap]
-version = ">= 1.6, < 1.8"
+version = "1.6"
 optional = true
 
 [dependencies.indoc]
-version = "0.3.6"
+version = "1.0.3"
 optional = true
 
 [dependencies.inventory]
@@ -135,7 +99,7 @@
 version = "0.11.0"
 
 [dependencies.paste]
-version = "0.1.18"
+version = "1.0"
 optional = true
 
 [dependencies.pyo3-macros]
@@ -154,16 +118,13 @@
 version = "1.1.0"
 
 [dev-dependencies.bitflags]
-version = "=1.2.1"
+version = "1.2.1"
 
 [dev-dependencies.clap]
-version = "=2.33"
-
-[dev-dependencies.criterion]
-version = "=0.3.4"
+version = "2.33"
 
 [dev-dependencies.half]
-version = "=1.7.1"
+version = "1.7.1"
 
 [dev-dependencies.proptest]
 version = "0.10.1"
