--- pyo3-0.16.6/Cargo.toml	1970-01-01T00:00:01+00:00
+++ pyo3-0.16.6/Cargo.toml	2023-01-17T19:53:56.296203+00:00
@@ -56,52 +56,6 @@
     "docsrs",
 ]
 
-[[bench]]
-name = "bench_call"
-harness = false
-
-[[bench]]
-name = "bench_err"
-harness = false
-
-[[bench]]
-name = "bench_dict"
-harness = false
-
-[[bench]]
-name = "bench_frompyobject"
-harness = false
-required-features = ["macros"]
-
-[[bench]]
-name = "bench_gil"
-harness = false
-
-[[bench]]
-name = "bench_list"
-harness = false
-
-[[bench]]
-name = "bench_pyclass"
-harness = false
-required-features = ["macros"]
-
-[[bench]]
-name = "bench_pyobject"
-harness = false
-
-[[bench]]
-name = "bench_set"
-harness = false
-
-[[bench]]
-name = "bench_tuple"
-harness = false
-
-[[bench]]
-name = "bench_intern"
-harness = false
-
 [dependencies.anyhow]
 version = "1.0"
 optional = true
@@ -118,7 +72,7 @@
 optional = true
 
 [dependencies.indexmap]
-version = ">= 1.6, < 1.8"
+version = "1.6"
 optional = true
 
 [dependencies.indoc]
@@ -161,9 +115,6 @@
 [dev-dependencies.assert_approx_eq]
 version = "1.1.0"
 
-[dev-dependencies.criterion]
-version = "0.3.5"
-
 [dev-dependencies.proptest]
 version = "0.10.1"
 features = ["std"]
@@ -171,9 +122,6 @@
 
 [dev-dependencies.rustversion]
 version = "1.0"
-
-[dev-dependencies.send_wrapper]
-version = "0.5"
 
 [dev-dependencies.serde]
 version = "1.0"
@@ -184,9 +132,6 @@
 
 [dev-dependencies.trybuild]
 version = "1.0.49"
-
-[dev-dependencies.widestring]
-version = "0.5.1"
 
 [build-dependencies.pyo3-build-config]
 version = "0.16.6"
@@ -236,8 +181,6 @@
     "eyre",
     "anyhow",
 ]
-generate-abi3-import-lib = ["generate-import-lib"]
-generate-import-lib = ["pyo3-ffi/generate-import-lib"]
 macros = [
     "pyo3-macros",
     "indoc",
