--- pyo3-0.18.1/Cargo.toml	1970-01-01T00:00:01+00:00
+++ pyo3-0.18.1/Cargo.toml	2023-03-03T19:29:34.899195+00:00
@@ -57,52 +57,6 @@
     "docsrs",
 ]
 
-[[bench]]
-name = "bench_call"
-harness = false
-
-[[bench]]
-name = "bench_err"
-harness = false
-
-[[bench]]
-name = "bench_dict"
-harness = false
-
-[[bench]]
-name = "bench_frompyobject"
-harness = false
-required-features = ["macros"]
-
-[[bench]]
-name = "bench_gil"
-harness = false
-
-[[bench]]
-name = "bench_list"
-harness = false
-
-[[bench]]
-name = "bench_pyclass"
-harness = false
-required-features = ["macros"]
-
-[[bench]]
-name = "bench_pyobject"
-harness = false
-
-[[bench]]
-name = "bench_set"
-harness = false
-
-[[bench]]
-name = "bench_tuple"
-harness = false
-
-[[bench]]
-name = "bench_intern"
-harness = false
-
 [dependencies.anyhow]
 version = "1.0"
 optional = true
@@ -171,9 +125,6 @@
 
 [dev-dependencies.chrono]
 version = "0.4"
-
-[dev-dependencies.criterion]
-version = "0.3.5"
 
 [dev-dependencies.proptest]
 version = "0.10.1"
@@ -186,9 +137,6 @@
 [dev-dependencies.rustversion]
 version = "1.0"
 
-[dev-dependencies.send_wrapper]
-version = "0.6"
-
 [dev-dependencies.serde]
 version = "1.0"
 features = ["derive"]
@@ -198,9 +146,6 @@
 
 [dev-dependencies.trybuild]
 version = ">=1.0.70"
-
-[dev-dependencies.widestring]
-version = "0.5.1"
 
 [build-dependencies.pyo3-build-config]
 version = "0.18.1"
@@ -253,7 +198,6 @@
     "anyhow",
     "experimental-inspect",
 ]
-generate-import-lib = ["pyo3-ffi/generate-import-lib"]
 macros = [
     "pyo3-macros",
     "indoc",
